tests/case_sensitive/run.sh

```diff
@@ -106,10 +107,11 @@ function run() {
 trap cleanup_process EXIT
 trap "cleanup_data Upper_DB Upper_DB1 lower_db UPPER_DB_ROUTE sync_diff_inspector" EXIT
 
-# cleanup_data all_mode
 # also cleanup dm processes in case of last run failed
 cleanup_process $*
 cleanup_data Upper_DB Upper_DB1 lower_db UPPER_DB_ROUTE
-run $*
+run
+cleanup_data Upper_DB Upper_DB1 lower_db UPPER_DB_ROUTE
```

Comment:
seems we don't cleanup data after the run in other tests (I don't know why but just keep this behaviour)